portfolio_project1.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang='en'>
+<head>
+    <meta charset='UTF-8'>
+	<meta name='viewport' content='width=device-width, initial-scale=1.0'>
+    <title>Jake Rooms - Project 1</title>
+	<link rel='stylesheet' href='normalize.css'/>
+	<link rel='stylesheet' href='portfolio.css'/>
+</head>
+<body>
+	<!--Html for the top navigation bar-->
+	<nav class='link_bar'>
+		<a class='top_link' href='portfolio_home.html'>Home</a>
+	</nav>
+	<!--Html for the title-->
+	<div class='center_bar'>
+		<h1>Project 1</h1>
+	</div>
+	<!--Html for section 1 with text on the left-->
+	<section class ='paragraph'>
+		<h3>Event 1</h3>
+		<!--Flexbox for the text and image, changes to column-reverse after a media queery-->
+		<div class='text_box left'>
+		<p>"On the other hand, we denounce with righteous indignation and dislike men who are so beguiled and demoralized by the charms of pleasure of the moment, so blinded by desire, that they cannot foresee the pain and trouble that are bound to ensue; and equal blame belongs to those who fail in their duty through weakness of will, which is the same as saying through shrinking from toil and pain. These cases are perfectly simple and easy to distinguish. In a free hour, when our power of choice is untrammelled and when nothing prevents our being able to do what we like best, every pleasure is to be welcomed and every pain avoided. But in certain circumstances and owing to the claims of duty or the obligations of business it will frequently occur that pleasures have to be repudiated and annoyances accepted. The wise man therefore always holds in these matters to this principle of selection: he rejects pleasures to secure other greater pleasures, or else he endures pains to avoid worse pains."</p>
```

Comment:
<img width="175" alt="Screenshot 2021-09-09 at 11 28 20" src="https://user-images.githubusercontent.com/89837220/132670108-a4d11571-2b8e-4bb7-95ef-131ce8176433.png">
Different " ' ? Uncertain if relevant for content vs selectors

Reply:
just pasted from lorem ipsum, it's content so i think it's fine